tools/src/test/java/org/apache/kafka/tools/consumer/group/SaslClientsWithInvalidCredentialsTest.java

```diff
@@ -149,7 +149,7 @@ public void testConsumerGroupServiceWithAuthenticationFailure(String quorum, Str
     }
 
     @ParameterizedTest(name = "{displayName}.quorum={0}.groupProtocol={1}")
```

Comment:
Line 137 states that we can't use `TestInfoUtils.TestWithParameterizedQuorumName()` in the `@ParameterizedTest` annotation. Does it make sense to add that here, too?

Reply:
Interesting, I'm not very sure about this one 🤔

Reply:
@frankvicky Could you please add the following comment to this test case?
```
// NOTE: Not able to refer TestInfoUtils#TestWithParameterizedQuorumName() in the ParameterizedTest name.
```